.tekton/run-acceptance-test.yaml

```diff
@@ -0,0 +1,146 @@
+    pipelinesascode.tekton.dev/task-32: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/get-operator-files-unofficial-builds.yaml"		
+    pipelinesascode.tekton.dev/task-33: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-operator-tests.yaml"
+    pipelinesascode.tekton.dev/task-34: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/install-cluster.yaml"					
+    pipelinesascode.tekton.dev/task-35: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-pac-tests.yaml"
+    pipelinesascode.tekton.dev/task-36: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/install-serverless-operator.yaml"			
+    pipelinesascode.tekton.dev/task-37: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-pipelines-tests.yaml"
+    pipelinesascode.tekton.dev/task-38: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mini-must-gather.yaml"					
+    pipelinesascode.tekton.dev/task-39: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-triggers-tests.yaml"
+    pipelinesascode.tekton.dev/task-40: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mirror-and-tag-imagestreams.yaml"			
+    pipelinesascode.tekton.dev/task-41: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/verify-binaries-and-sha256sum.yaml"
+    pipelinesascode.tekton.dev/task-42: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mirror-operator-images.yaml"				
+    pipelinesascode.tekton.dev/task-43: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/verify-binaries-on-mac.yaml"
+    pipelinesascode.tekton.dev/task-44: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/release-tests.yaml"					
+    pipelinesascode.tekton.dev/task-45: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/whitelist-domains.yaml"
+    pipelinesascode.tekton.dev/task-46: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/run-tfa-classifire.yaml"
+    pipelinesascode.tekton.dev/max-keep-runs: "5"  
+spec:
+  params:
+    - name: repo_url
+      value: "{{ repo_url }}"
+    - name: revision
+      value: "{{ revision }}"
+  pipelineSpec:
+    params:
+      - name: repo_url
+      - name: revision
+    workspaces:
+      - name: source
+      - name: basic-auth
+      - name: cache
+    tasks:
+      - name: fetch-repository
+        taskRef:
+          kind: ClusterTask
+          name: git-clone
+        workspaces:
+          - name: output
+            workspace: source
+          - name: basic-auth
+            workspace: basic-auth
+        params:
+          - name: url
+            value: $(params.repo_url)
+          - name: revision
+            value: $(params.revision)
+        taskSpec:
+          pipelineRef:
+            name: acceptance-tests
+          params:
+            - name: ARCH
```

Comment:
You need to get release-tests fork's URL and revision into the acceptance-tests... that feature doesn't exist so it needs to be implemented first.